spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -293,6 +293,8 @@ object SpdxDeclaredLicenseMapping {
         "GNU General Public License (GPL) v. 3" to GPL_3_0_ONLY.toExpression(),
         "GNU General Public License, Version 2 with the Classpath Exception"
                 to (GPL_2_0_ONLY with CLASSPATH_EXCEPTION_2_0),
+        "GNU General Public License, version 2 with the GNU Classpath Exception"
+                to (GPL_2_0_OR_LATER with CLASSPATH_EXCEPTION_2_0),
```

Comment:
So, why is this actually `GPL_2_0_OR_LATER` instead of `GPL_2_0_ONLY`?

Reply:
Good finding, fixed it.